ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -76,6 +76,53 @@ tests:
         NUM_MASTERS=1
         OPENSHIFT_INSTALL_RELEASE_IMAGE=registry.svc.ci.openshift.org/sno-dev/openshift-bip:0.2.0
     workflow: baremetalds-assisted
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted-static-ip-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    env:
+      ASSISTED_CONFIG: |
+        ADDITIONAL_PARAMS="--with-static-ips"
+    workflow: baremetalds-assisted
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted-single-node-static-ip-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    env:
+      ASSISTED_CONFIG: |
+        NUM_MASTERS=1
+        ADDITIONAL_PARAMS="--with-static-ips"
+        OPENSHIFT_INSTALL_RELEASE_IMAGE=registry.svc.ci.openshift.org/sno-dev/openshift-bip:0.2.0
+    workflow: baremetalds-assisted
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted-single-node-ipv6-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    env:
+      ASSISTED_CONFIG: |
+        NUM_MASTERS=1
+        IPv4=no
+        IPv6=yes
+        PROXY=yes
+        OPENSHIFT_INSTALL_RELEASE_IMAGE=registry.svc.ci.openshift.org/sno-dev/openshift-bip:0.2.0
+    workflow: baremetalds-assisted
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted-openshift-48-milestone-one-periodic
```

Comment:
How is this one different from the previous one?

Reply:
It's using static IP

Reply:
OK I missed that. I think you should name it with its priorities.

Reply:
What do you mean?

Reply:
Like the rest. `e2e-metal-assisted-single-node-ipv6-static-ips-periodic`
But should static IPs have their own test? Isn't that a feature that should be covered by a none-platform test?

Reply:
milestone 1 will be appended very soon with more features, so I don't think we should append all it's features in its name :)
I also think we can be better if we have separate tests for features like static IP, so we can understand how they behave in isolation
And we do need the full flow, because static IPs handling depends very much on providing the right ignitions/manifests to MCO's services